adflow/pyADflow.py

```diff
@@ -5460,6 +5715,40 @@ def _readPlot3DSurfFile(self, fileName, convertToTris=True):
 
         return pts, conn
 
+    # mham
```

Comment:
these are code and comments that are useful to perform a dot product test of the newly added AD routines. You should probably perform a dot product test, and if they work, you can move this routine in a separate branch, or save them as notes for developers etc, but I dont think they should stay in pyadflow